script/UpgradeAllBridge.s.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+import { UpdateScriptBase } from "./utils/UpdateScriptBase.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+import { DiamondCutFacet, IDiamondCut } from "lifi/Facets/DiamondCutFacet.sol";
+import { AllBridgeFacet } from "lifi/Facets/AllBridgeFacet.sol";
+
+contract DeployScript is UpdateScriptBase {
+    using stdJson for string;
+
+    function run() public returns (address[] memory facets) {
+        address facet = json.readAddress(".AllBridgeFacet");
+
+        vm.startBroadcast(deployerPrivateKey);
+
+        // Across
```

Comment:
Across => AllBridge (or remove entirely since the comment doesnt really say anything

Reply:
rm